<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>旅行计划</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="Schedule">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newSchedule">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新建日程</string>
   </property>
  </widget>
  <widget class="QListWidget" name="TripPlan">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newPlan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新建计划</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteSchedule">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除日程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePlan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除计划</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
